Packs/CVE_2025_31324/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2835 @@
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3637.25,
+          "y": -553.5
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "115":
+    id: "115"
+    taskid: 830bb98c-ec2a-4fb5-840d-fc37a5590c7f
+    type: regular
+    task:
+      id: 830bb98c-ec2a-4fb5-840d-fc37a5590c7f
+      version: -1
+      name: Tag CVE indicators
+      description: commands.local.cmd.new.indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "117"
+    scriptarguments:
+      retry-count:
+        simple: "3"
+      retry-interval:
+        simple: "2"
+      tags:
+        simple: SAP NetWeaver's Visual Composer Framework
+      type:
+        simple: CVE
+      value:
+        complex:
+          root: ExtractedIndicators
+          accessor: CVE
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4031.25,
+          "y": -553.5
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "116":
+    id: "116"
+    taskid: ab7b0e86-8449-40d0-8575-ccba7356c050
+    type: title
+    task:
+      id: ab7b0e86-8449-40d0-8575-ccba7356c050
+      version: -1
+      name: Extract Indicators
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "112"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4225,
+          "y": -962.5
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "117":
+    id: "117"
+    taskid: 9960cbb4-59cb-4e84-8d9a-cd6bb507e77f
+    type: regular
+    task:
+      id: 9960cbb4-59cb-4e84-8d9a-cd6bb507e77f
+      version: -1
+      name: Link indicators to alert
+      description: commands.local.cmd.associate.indicators
+      script: Builtin|||associateIndicatorsToIncident
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "108"
+    scriptarguments:
+      incidentId:
+        simple: ${incident.id}
+      indicatorsValues:
+        complex:
+          root: ExtractedIndicators.IP
+          filters:
+          - - operator: isNotEmpty
+              left:
+                value:
+                  simple: ExtractedIndicators.IP
+                iscontext: true
+          transformers:
+          - operator: append
```

Comment:
appaned if not empty should be better here (for all)
or adding removeEmpty transformer